prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<head>
+  <title>Contact Us</title>
+</head>
+  <h1>Contact Us</h1>
+  <a href="/prep-html-link/index.html">Click here to go to the home page</a>
+  <link href="prep-html-link/index" rel="contact-us/index">
```

Comment:
Hi Gabrielle, be sure to put the visible content of your page into a `body` element and the `head` and `body` both belong in an `html` element, as outlined in the `prep-html-skeleton` exercise.